src/music/element/MusicQueue.py

```diff
@@ -9,8 +9,11 @@
 class MusicQueue:
     """Reperesents a Queue GUI object. Handles which MusicSource to play next and displays in the GUI.
     """
-    def __init__(self, active_embed: ChatEmbed, client: discord.Client, list: List[MusicSource] = []) -> None:
-        self.list = list
+    def __init__(self, active_embed: ChatEmbed, client: discord.Client, playlist: List[MusicSource] = None) -> None:
```

Comment:
Any reason why you had to set playlist to None in the arguments instead of just having it set as an empty list by default

Reply:
That was the problem that I fixed when servers were sharing queue's (list variable) with each other since python is strictly pass by reference only. When I initialized a queue, the default behavior was to use the same empty list address for all the queues for all the servers. From what I researched default variables in classes are counted as class variables, and anything inside the function is counted as instance variables.

Reply:
gotcha, that makes sense